src/app/child-dev-project/attendance/model/attendance-status.ts

```diff
@@ -0,0 +1,8 @@
+export enum AttendanceStatus {
```

Comment:
candidate for config?

Reply:
yes, I was thinking the same. just a bit intimidated by the services and classes required to set it up. Let's work on this in a separate PR in a pair programming session while possibly simplifying the general process.